examples/passwords.yml

```diff
@@ -10,3 +10,5 @@
 # $ ansible-playbook myplaybook.yml --ask-vault-pass
 #
 engine_password: 123456
+username: "myuser"
+password: "mypass"
```

Comment:
Wouldn't it be more understandable to use  `ovirt_repositories_rh_username`/ `ovirt_repositories_rh_password` instead of `username`/`password`?

Reply:
Fixed.